Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "nng-sys"
-version = "1.1.1-rc.1"
+name = "runng-sys"
```

Comment:
It looks like this got accidentally renamed.

Reply:
It was intentional since I've actually been using the crate, but didn't need to be in the PR.